cea/demand/thermal_loads.py

```diff
@@ -1075,7 +1054,7 @@ def lookup_specific_heat_capacity(self, th_mass):
         else:
             return 165000.0
 
-    def lookup_effective_mass_area_factor(self, th_mass):
+    def lookup_effective_mass_area_factor(self, cm):
         """
         Look up the factor to multiply the conditioned floor area by to get the effective mass area by building construction
```

Comment:
this would be a good place to point out where in the standards this comes from...